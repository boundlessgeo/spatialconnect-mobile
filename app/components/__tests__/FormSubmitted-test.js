/*global jest,describe,it,expect*/
import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import FormSubmitted from '../FormSubmitted';

describe('<FormSubmitted />', () => {

  it('renders correctly', () => {
    const tree = renderer.create(
      <FormSubmitted />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

});