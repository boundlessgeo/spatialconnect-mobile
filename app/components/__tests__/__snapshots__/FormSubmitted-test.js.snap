exports[`<FormSubmitted /> renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#e4e7ec",
      "flex": 1,
      "marginTop": 0,
      "padding": 10
    }
  }>
  <Text
    style={
      Object {
        "fontWeight": "bold",
        "marginBottom": 5,
        "marginTop": 5
      }
    }>
    Form submitted.
  </Text>
</View>
`;
